src/main/scala/io/github/davidgregory084/ScalacOptions.scala

```diff
@@ -29,7 +29,11 @@ private[davidgregory084] trait ScalacOptions {
 
   /** Provide another option that is not declared by the ScalacOptions DSL.
     */
-  def other(option: String, args: List[String], isSupported: ScalaVersion => Boolean): ScalacOption =
+  def other(
+    option: String,
+    args: List[String],
```

Comment:
Also noticing here that the naming is not in line with previously existing `privateOption`

Reply:
Hmm do you mean the argument names? This is because `def other` was added after the `ScalacOption` breaking changes in 0.4.0.

Reply:
yes, the argument name is not the same. It was named `arguments` in `privateOption`.
This isn't a problem at all. Just raising a inconsistency :)

Reply:
Yes it's a good point, I wonder if we could use https://www.scala-lang.org/api/2.13.5/scala/deprecatedName.html to bring them all in line